src/App.js

```diff
@@ -0,0 +1,51 @@
+import {
+  BrowserRouter as Router, Routes, Route,
+} from 'react-router-dom';
+import Cookies from 'js-cookie';
+import { useDispatch } from 'react-redux';
+import { useEffect } from 'react';
+import SplashPage from './components/SplashPage';
+import ProtectedRoute from './components/ProtectedRoute';
+import { loginAsync } from './redux/users/userSlice';
+import Doctors from './components/Doctors';
+import Layout from './components/Layout';
+import Details from './components/Details';
+import Appointments from './components/Appointments';
+import NewAppointment from './components/NewAppointment';
+import NewPasswordForm from './authentication/passwords/newPassword';
+import NewDoctor from './Pages/NewDoctor';
+import DeleteDoctor from './components/delete';
+
+function App() {
```

Comment:
- [ ] When trying to register a new user locally, since the app is setup to consume the cloud API and not the local, I get a CORS error, because the server is set to only accept requests from onrender:

![image](https://github.com/zeff96/doctor_appoitment_front_end/assets/57421823/7f361f74-a2ad-4c0c-86f7-4fab8fe2ce71)

![image](https://github.com/zeff96/doctor_appoitment_front_end/assets/57421823/3d370f1d-970e-41ee-a2bd-6dea4ae7618c)

Even if I set both URLs in userSlice and doctorSlice to http://127.0.0.1:3000, and change the origins in CORS to local or *, it does not work.

Kindly, provide instructions and/or set your project to be able to be tested locally. 👍 

_note: Since we are required to review the cloned project locally, this means I was not able to test the app's functionality.  This will be taken a look at in the next review._